src/CareerBuilder/OAuth2/Flows/Flow.php

```diff
@@ -118,10 +144,13 @@ public function getToken(AccessToken $token = null)
             $this->buildBody();
         }
 
-        $request = $this->client->post($this->tokenRequestPath, $this->headers, $this->body);
-        $response = $request->send();
-        $data = $response->json();
-
+        /** @var Response $response */
+        $response = $this->client->post($this->tokenRequestPath,[
+            'headers' => $this->headers,
+            'form_params' =>$this->body
```

Comment:
❌Hey, this code is not following the PSR-12 standard, which we are trying to strongly follow in this codebase.
Doc: https://www.php-fig.org/psr/psr-12